react/__mocks__/vtex.styleguide.tsx

```diff
@@ -1,9 +1,9 @@
-import React from 'react'
+import React, { FunctionComponent } from 'react'
 
-export const Button = ({ children = undefined, onClick = () => {} }) => (
-  <button type="button" onClick={onClick}>
-    {children}
+export const Button: FunctionComponent = (props: any) => (
```

Comment:
I like the previous version better, was this done bc of types?

Reply:
same for similar changes on the other files.

using `any` kind of defeats the purpose of using types in the first place. could you set the types of each prop?